library-best-practices/JLBP-5.md

```diff
@@ -12,31 +12,33 @@ This can happen in several ways:
 
 * A third party forks a library under their own group ID but does not repackage the classes.
 
-* A third library copies an existing library's packages
+* A third party library copies an existing library's packages
 into its own jar file without shading the dependency. This case is particularly
 insidous since it may not be obvious that there's an unexpected, undocumented
 version of the classes hiding inside the seemingly unrelated jar.
 
-Example 1: Guava's main artifact is `guava`, but from versions 13.0 to 17.0,
+Example 1: Guava's main artifact ID is `guava`, but from versions 13.0 to 17.0,
   another artifact `guava-jdk5` was also published with classes that overlap
-  with `guava`. Build systems such as Maven and Gradle can't dedupe `guava-jdk5`
-  with `guava` because the artifact names are different, which means that the
-  same class names are provided by multiple jars, and runtime errors result from
-  classes and methods not being found. The only known Google library propagating
-  this dependency is google-api-client <= 1.23.0, and it has removed the
+  with `guava`. Build systems such as Maven and Gradle cannot deduplicate
```

Comment:
For talking about past experiences, past tense sounds natural for me, as it already did in line 21.